cmd/webhook/main.go

```diff
@@ -45,6 +46,7 @@ import (
 	// config validation constructors
 	tracingconfig "knative.dev/pkg/tracing/config"
 	defaultconfig "knative.dev/serving/pkg/apis/config"
+	configvalidation "knative.dev/serving/pkg/apis/config/validation"
```

Comment:
This is sort of an odd place to put this.  Can we make it `knative.dev/serving/pkg/leaderelection`?

Reply:
Note: the defaulting config is here because defaulting happens as part of the pkg/apis stuff

Reply:
That's fair. I had something similar in eventing that already went in, mind if we do this in a follow-up?